modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/login-button/index.html

```diff
@@ -0,0 +1,3 @@
+<div class="align-items-end d-flex evp-login-button justify-content-center">
+<a href="/c/portal/login" type="button" class="btn" id="evp-login-button">Sign in</a>
```

Comment:
1. Spacing (TAB)
2. Order the attributes, `class` come first, then `href` ...
3. No need for this ID

```suggestion
    <a href="/c/portal/login" type="button" class="btn">Sign in</a>
```